ui/externs/watermark.js

```diff
@@ -0,0 +1,47 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/**
+ * @externs
+ * @suppress {duplicate} To prevent compiler errors with the namespace
+ *   being declared both here and by goog.provide in the library.
+ */
+
+
+/**
+ * @typedef {{
+ *   type: string,
+ *   text: string,
+ *   position: string,
+ *   color: string,
+ *   size: number,
+ *   alpha: number,
+ *   interval: number,
+ *   skip: number
+ * }}
+ *
+ * @property {string} type
+ *   The type of watermark ('static' or 'dynamic'). Defaults to 'static'.
```

Comment:
```suggestion
 *   The type of watermark ('static' or 'dynamic'). 
 *   <br>
 *   Defaults to 'static'.
```

Add a br in all defaults